package-res/ccc/core/base/chart/chart.panels.js

```diff
@@ -64,7 +64,8 @@ pvc.BaseChart
         this._initTitlePanel();
 
         // null on small charts or when not enabled
-        var legendPanel = this._initLegendPanel(),
+        var preserve = this._preserveLayout,
```

Comment:
The added variable left the above comment out of place as it referred to the legendPanel variable. Coincidently, the `preserve` variable deserved a similar comment informing that it is only ever true for leaf-charts (not a multi-chart root).
In fact, its value is only used inside the `if(!isMultichartRoot)`, and so the variable declaration and initialization could be beneficially moved to inside the "if".